cfme/automate/explorer/method.py

```diff
@@ -51,22 +53,98 @@ def is_displayed(self):
             self.fqdn.text == self.context['object'].tree_path_name_only[1:])
 
 
+class PlaybookBootstrapSelect(BootstrapSelect):
+    """BootstrapSelect widget for Ansible Playbook Method form.
+
+    BootstrapSelect widgets don't have `data-id` attribute in this form, so we have to override ROOT
+    locator.
+
+    """
+    ROOT = ParametrizedLocator('.//select[normalize-space(@name)={@id|quote}]/..')
+
+
+class ActionsCell(View):
+    edit = Button(**{"ng-click": "vm.editKeyValue(this.arr[0], this.arr[1], this.arr[2], $index)"})
```

Comment:
**QUESTION**: Does this work? I had instances where this trick did not work and I don't remember where it was. But I have a PR for wt.pf that will translate underscores into dashes, so that should solve the issue in case it pops up

Reply:
```
In [2]: (lambda **k: k)(**{'-': True})
Out[2]: {'-': True}
```

in my test works on both python2 and python3